src/components v2/Molecules/Forms/FormControl/FormControl.vue

```diff
@@ -68,9 +68,13 @@ export default Vue.extend({
       default: '',
     },
     size: {
-      type: String as PropType<string>,
```

Comment:
We remove the `PropType` on purpose?

Reply:
Not intentional. Needs to be a PropType :D.

This is why we have you Nishimura-Senpai